src/Robo/Plugin/Commands/VrtRunCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Fire\Robo\Plugin\Commands;
+
+use Robo\Symfony\ConsoleIO;
+use Robo\Robo;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Provides a command to run the VRT testing.
+ */
+class VrtRunCommand extends FireCommandBase {
+
+  /**
+   * Runs your VRT testing (lando Only)
+   *
+   * Usage Example: fire vrt-run
+   *
+   * @command vrt:run
+   * @aliases vrun
+   *
+   */
+  public function vrtRun(ConsoleIO $io) {
+    $env = Robo::config()->get('local_environment');
+    $tasks = $this->collectionBuilder($io);
+    if ($env == 'lando') {
+      $reconfigureTestingUrls = $io->choice('Do you want yo reconfigure your reference and test urls?', ['Yes', 'No']);
+      $newReferenceFiles = $io->choice('Do you want to re-take the reference screenshots?', ['Yes', 'No']);
+      if (strtolower($reconfigureTestingUrls) === 'yes') {
+        $tasks->addTask($this->taskExec('fire vrt:testing-setup'));
+      }
+      if (strtolower($newReferenceFiles) === 'yes') {
```

Comment:
I wonder if these yes/no questions should use the [confirm()](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Console/Style/SymfonyStyle.php#L229) method? 

```suggestion
      $reconfigureTestingUrls = $io->confirm('Do you want to reconfigure your reference and test urls?', true);
      $newReferenceFiles = $io->confirm('Do you want to re-take the reference screenshots?', true);
      if ($reconfigureTestingUrls) {
        $tasks->addTask($this->taskExec('fire vrt:testing-setup'));
      }
      if ($newReferenceFiles) {
```

Reply:
Great suggestion, I was not of aware of its existince , thank you!